docs/pages/deployment/discovery.rst

```diff
@@ -24,6 +24,50 @@ Service definitions are JSON files loaded from the ``discovery.definitions.direc
 It loads all files wih the ``.json`` extension in this directory. It does not load subdirectories.
 If the directory contains JSON files that are not (valid) service definitions, the node will fail to start.
 
+Discovery Server
+================
+
 To act as server for a specific discovery service definition,
 the service ID from the definition needs to be specified in ``discovery.server.defition_ids``.
-The IDs in this list must correspond to the ``id`` fields of the loaded service definition, otherwise the node will fail to start.
\ No newline at end of file
+The IDs in this list must correspond to the ``id`` fields of the loaded service definition, otherwise the node will fail to start.
+
+Discovery Client
+================
+
+Clients update their local copy of the Discovery Service at a set interval, this can be configured with ``discovery.client.refresh_interval``.
+
+Usage
+*****
+
+Applications need to integrate with the Discovery Service to make their organization's discoverable for a use case,
+and/or to find other organizations supporting a certain use case.
+
+Registration
+============
+
+To make an organization discoverable through a Discovery Service, need to activate the organization's DID on that particular service.
+This can be done by performing a HTTP POST to the ``/internal/discovery/v1/{serviceID}/{did}`` endpoint of the Nuts node,
+where the service ID must match the ID of the service definition.
+The Nuts node will try to create a Verifiable Presentation according to the Service Definition and register it on the Discovery Server.
+
+Registration on the Discovery Service will only succeed if then the DID's wallet contains the required credentials.
+If registration fails it will be retried after a set interval, which can be configured with ``discovery.client.registration_refresh_interval``.
```

Comment:
```suggestion
If registration fails, it will be retried after a set interval, which can be configured with ``discovery.client.registration_refresh_interval``.
```